src/webgpu/api/operation/buffers/map_oom.spec.ts

```diff
@@ -32,9 +32,10 @@ g.test('mapAsync')
     oomAndSizeParams //
       .beginSubcases()
       .combine('write', [false, true])
+      .combine('abortError', [false, true])
```

Comment:
nit: unmapBeforeResolve ? It makes it more clear what the test checks.

Reply:
Thanks for the suggestion. Renamed to clarify, thanks.